osmaxx/excerptexport/static/excerptexport/scripts/maps/draw_controls.js

```diff
@@ -82,6 +89,41 @@ var draw_controls = function (map) {
             nameField.parentNode.appendChild(e);
         }
 
+        function getAllowedMaxSize(){
+            var selectedDetailLevel = jQuery('#id_detail_level');
```

Comment:
As you've already introduced that variable, you could use `detailLevelID` in place of `'#id_detail_level'`.